### PR TITLE
Add remind command

### LINKED ISSUE #101: Add remind command

### PR DESCRIPTION
**Pull request by jannaleong**
Fixes #101

### CODE REVIEW

File: src/main/java/seedu/address/logic/messages/NoteMessages.java
Comment (yleeyilin) [inline src/main/java/seedu/address/logic/messages/NoteMessages.java:28]: Use maintainer here, we are standardising to maintainer.

**Review by yleeyilin — COMMENTED** [review]
Minor issue, will approve after rectification. Otherwise, looks good!

File: src/main/java/seedu/address/logic/messages/NoteMessages.java
Comment (jannaleong) [reply src/main/java/seedu/address/logic/messages/NoteMessages.java]: Noted!! i have made the changes.

**Review by yleeyilin — APPROVED** [review]
LGTM!

**Review by jamessinmaojun — APPROVED** [review]
lgtm

**Pull request merged**
